### PR TITLE
Add benchmarking info to README

### PR DESCRIPTION
**Pull request by emjin**
PR checklist:
- [x] changelog is up to date

### CODE REVIEW

**Review by aryx — APPROVED** [review]
Good

File: perf/run-benchmarks [hunk 1]
```diff
@@ -390,6 +398,9 @@ def main() -> None:
     parser.add_argument(
         "--upload", help="upload results to semgrep dashboard", action="store_true"
     )
+    parser.add_argument(
+        "--std_only", help="only run the default semgrep", action="store_true"
```
Comment (mjambon) [inline perf/run-benchmarks:402]: I'd like to change option flags to using only dashes (`--std-only`, `--plot-benchmarks`), as it is most common and expected by users.

Comment (emjin) [reply perf/run-benchmarks:402]: Will do! Our options flags are somewhat disorganized; I'm thinking about separating the corpuses into small (< 10 s), medium (< 2 min), and large (rest), and having corresponding flags

Comment (aryx) [reply perf/run-benchmarks:402]: I recently read that the fact that we use -- for flags is an accident; originally short was supposed to use -x and long argument +xxx :)

**Pull request merged**